### PR TITLE
ddl: fix TestModifyColumnRollBack

### LINKED ISSUE #15639: db_test.go:testDBSuite5.TestModifyGeneratedColumn failed
db_test.go:testDBSuite5.TestModifyGeneratedColumn

Latest failed builds: 
https://internal.pingcap.net/idc-jenkins/job/tidb_ghpr_unit_test/28796/display/redirect

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What problem does this PR solve?

Issue Number: close #15639 

Problem Summary:
If `insert into t1(c2) values (null);` runs firstly, the error will be `[ddl:1265]Data truncated for column 'c2' at row 1` instead of `[ddl:8214]Cancelled DDL job`.

### What is changed and how it works?
Put ``insert into t1(c2) values (null);`` after loop where the DDL is canceled.

### Check List 

Tests 

- No code

Side effects

### CODE REVIEW

File: ddl/db_test.go [hunk 1]
```diff
@@ -3292,19 +3292,16 @@ func (s *testDBSuite5) TestModifyColumnRollBack(c *C) {
...
 	go backgroundExec(s.store, "alter table t1 change c2 c2 bigint not null;", done)
-	ticker := time.NewTicker(s.lease / 2)
-	defer ticker.Stop()
 LOOP:
 	for {
 		select {
 		case err := <-done:
 			c.Assert(err, NotNil)
 			c.Assert(err.Error(), Equals, "[ddl:8214]Cancelled DDL job")
 			break LOOP
```
[the remark this answers is not in the document]
Comment (crazycs520) [reply ddl/db_test.go:3301]: Great catch. +1

**Comment by zz-jason** [comment]
@wjhuang2016 should we cherry-pick this pr to other release branches?

**Pull request merged**